stable/stolon/values.yaml

```diff
@@ -132,3 +132,21 @@ sentinel:
   podDisruptionBudget:
     # minAvailable: 1
     # maxUnavailable: 1
+
+## initdb scripts
+## Specify dictionary of scripts to be run at first boot, the entry point script is create_script.sh
+## i.e. you can use pgsql to run sql script on the cluster.
+##
+# initdbScripts:
+#   create_script.sh: |
+#      #!/bin/sh
+#      echo "Do something."
+
+## nodePostStart scripts
+## Specify dictionary of scripts to be run at first boot, the entry point script is postStartScript.sh
+## i.e. you can create tablespace directory here.
+##
+# nodePostStartScript:
+#  postStartScript.sh: |-
```

Comment:
Why there need a `|-`  ?  The scripts don't need to remove the blank in the end right?

Reply:
You are right. Fixed